src/modules/partition/gui/ChoicePage.cpp

```diff
@@ -679,7 +679,11 @@ ChoicePage::onLeave()
 {
     if ( m_config->installChoice() == InstallChoice::Alongside )
     {
-        doAlongsideApply();
+        if ( m_afterPartitionSplitterWidget->splitPartitionSize() >= 0
+             && m_afterPartitionSplitterWidget->newPartitionSize() >= 0 )
+        {
+            doAlongsideApply();
+        }
```

Comment:
I think it would be good to log something in the `else` case, so we (or the user) can know what happened, anyway.